test/utils/utils.test.js

```diff
@@ -71,19 +76,19 @@ describe('Utils', () => {
   });
 
   it('Converts Base 64 to UTF-8', () => {
-    window.location.hash = hashConfig;
-    const config = utils.getHashConfig();
-    expect(config.cardStyle).to.equal('full-card');
+    window.location.hash = hash;
+    const hashConfig = utils.getHashConfig();
+    expect(hashConfig.cardStyle).to.equal('full-card');
   });
 
   it('Successfully dies parsing a bad config', () => {
     utils.parseEncodedConfig('error');
     expect(console.log.args[0][0].name).to.equal('InvalidCharacterError');
   });
 
-  it('Text getEnv()', () => {
-    expect(utils.getEnv()).to.equal('local');
-  });
+  // it('Test getEnv()', () => {
```

Comment:
getConfig instead now?

Reply:
Yup.

1. getEnv() should not be exported anymore.
2. The comments need to be removed, too.

Thank you!